bin/fin

```diff
@@ -5206,23 +5225,34 @@ config_remove ()
 		target_env_file="$CONFIG_ENV"
 	else
 		load_configuration
+		if [[ -n $env ]]; then
+			local docksal_env_file="$(get_project_path_dc)/.docksal/docksal-${env}.env"
+		fi
 		target_env_file="$docksal_env_file"
 	fi
 
-	while [ $# -gt 0 ]
+	# Warn if no values were found
+	if [[ "${#ARGV[@]}" == 0 ]]; then
+		echo "No values to remove"
+		return 1
+	fi
+
+	# Loop named params array
+	for param in "${ARGV[@]}"
 	do
-		if [[ "$1" == "--global" ]]; then
+		if [[ "${param}" =~ ^- ]]; then
```

Comment:
this condition is not needed

Reply:
@achekulaev what ends up happening is if I use `--env` it pics it up as a variable. If that is the case then I believe that `parse_params` may need to be looked at because when we set `--env=local` without this piece `$ARGV` is adding `--env` to the variable.

Reply:
```bash
§ fin debug parse_params --env=local TEST
local ARGV=(); local ARGN=(); local ARGO=();
ARGN+=('--env=local');
local env="local";
ARGV+=('TEST');
```

everything seems to be in order, `--env` does not get into ARGV.

Reply:
@achekulaev you are correct, I was getting the config_remove and the config_set module mixed up.